src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -82,6 +86,28 @@ private String getSanitizedConfig() {
   public void start(Promise<Void> startFuture) {
     log.debug("Startup configuration: {}", this::getSanitizedConfig);
 
+    HttpServer httpServer = vertx.createHttpServer();
+
+    httpServer.requestHandler(request -> {
+      log.info("path : {}",request.path());
+      if (request.path().equals("/admin/health")) {
+        HttpServerResponse response = request.response();
+        response.setStatusCode(200);
+        response.putHeader("Content-Type", "text/plain");
+        response.end("OK");
+        log.info("statusCode : {}",response.getStatusCode());
```

Comment:
Same comment for the log as put at line number 92

Reply:
Added